README.md

```diff
@@ -43,6 +44,11 @@ It is a standalone Microservice that connects the two cloud platforms and provid
 
 Read [Project lifecycle](docs/Project-Lifecycle.md) documentation for full information.
 
+## For shop integration (front/back end developers)
+
+Read [Payone Integration Service Shop Integration Guide](docs/Shop-Integration-Guide.md): for those developers, 
+who integrate the service with online shop.
+
```

Comment:
`List of TODOs which has to be covered as part of frontend integration.`

Reply:
sorry, this comment is not clear

Reply:
258e56a